cardano-db-sync/src/Cardano/DbSync/Cache/Types.hs

```diff
@@ -53,12 +60,13 @@ data CacheStatus
 
 data CacheAction
   = UpdateCache
+  | UpdateCacheStrong
```

Comment:
I was a little left confused with the naming of `UpdataCacheStong` from my understanding:

`UpdataCacheStong` inserts into `scStableCache`
`scStableCache` inserts into `scLruCache`

not sure if renaming would help, here's a couple:

```haskell
data CacheUpdateStrategy
  = UpdateLRU
  | UpdateStable
  | NoUpdate
  | EvictAndUpdate
  deriving (Eq)
```
```haskell
data CacheUpdateStrategy
  = UpdateFlexible
  | UpdateFixed
  | NoUpdate
  | EvictAndUpdate
  deriving (Eq)
```